package.json

```diff
@@ -65,7 +65,7 @@
     "identity-obj-proxy": "^3.0.0",
     "ignore-loader": "^0.1.2",
     "jest": "^22.4.3",
-    "node-sass": "^4.12.0",
+    "node-sass": "^6.0.1",
```

Comment:
does this version upgrade works with both Node versions 12 and 16

Reply:
Yes it works with both